src/Catel.MVVM/Catel.MVVM.Shared/MVVM/Converters/ValueConverterGroup.cs

```diff
@@ -35,13 +37,13 @@ namespace Catel.MVVM.Converters
     /// Original license: CPOL, compatible with the MIT license.
     /// </remarks>
     [ContentProperty("Converters")]
-    public class ValueConverterGroup : IValueConverter
+    public class ValueConverterGroup : System.Windows.Data.IValueConverter
```

Comment:
I think it's easier to write this once in the usings:

using IValueConverter = System.Windows.Data.IValueConverter;